### ISSUE TITLE
[BUG] Minimap error with LFG Bulletin Board

### ISSUE THREAD

**Issue by brutzel** [Karl-HeinzSchneider/WoW-DragonflightUI#292]
**Describe the bug**
When you have LFG Bulletin Board installed, you get this error after login or reload:

**Error Log**
```
Message: Interface/AddOns/DragonflightUI/Modules/Minimap.lua:1406: attempt to index local 'btn' (a nil value)
Time: Sun Dec 15 02:33:44 2024
Count: 1
Stack: Interface/AddOns/DragonflightUI/Modules/Minimap.lua:1406: attempt to index local 'btn' (a nil value)
[string "@Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:66: in function <...ce/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:523: in function `EnableAddon'
[string "@Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:536: in function `EnableAddon'
[string "@Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:626: in function <...ce/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:611>
```
Locals: 

**Versions (please complete the following information):**
 - WoW: Era
 - Addon 0.19.13 with LFG Bulletin Board 3.41

**Additional context**

**Comment by Karl-HeinzSchneider** [comment]
Can't confirm that. Button is there and styled, even with dark mode.
But just to make sure, I added a safeguard cause why not: 938ff0028bd10abb999990f23052e646a807a5b3
Now it should just be a visual problem; and if any other error is fixed, I could investigate more. I tried different locales and with other addons, but for me everything works as expected.

**Comment by brutzel** [comment]
The button is there, correct, but if I enable `console scriptErrors 1` I get the mentioned lua error after reload or at login.

**Comment by brutzel** [comment]
Can confirm this is fixed with 0.19.14. Thanks!
